### PR TITLE
[DOCS] Reload the library that could retain global gpu resources

### PR DESCRIPTION
**Pull request by tqchen**
cc @areusch @mbs-octoml

### CODE REVIEW

**Comment by tqchen** [comment]
background https://github.com/sphinx-gallery/sphinx-gallery/issues/853

**Review by junrushao1994 — APPROVED** [review]

**Comment by mbs-octoml** [comment]
I didn't have any luck using just the module name nor explicitly removing all tensorflow* modules as they do for 'seaborn' (the latter triggers an internal tensorflow error). I've also tried launching the tensorflow tutorial as a sub-process but that didn't seem to actually separate out the module loads.

File: docs/conf.py [hunk 1]
```diff
@@ -341,7 +339,7 @@ def force_gc(gallery_cong, fname):
     "download_all_examples": False,
     "min_reported_time": 60,
     "expected_failing_examples": [],
-    "reset_modules": (force_gc, "matplotlib", "seaborn"),
+    "reset_modules": (force_gc, "matplotlib", "tensorflow", "torch", "onnx"),
```
Comment (areusch) [inline docs/conf.py:342]: i don't see torch or onnx here: https://github.com/sphinx-gallery/sphinx-gallery/blob/b41e328230f016b2089464b8f834fd76a6395eac/sphinx_gallery/scrapers.py#L561

what am i missing?

**Comment by tqchen** [comment]
Good catch. Seems indeed this would need some special support from sphinx gallery side.

**Comment by tqchen** [comment]
closing for now as it is more complicated than I initially thought

**Pull request closed without merging**